boards/cc1312-launchpad/Makefile.include

```diff
@@ -8,4 +8,11 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
 include $(RIOTMAKE)/tools/serial.inc.mk
 
 # configure the flash tool
-include $(RIOTMAKE)/tools/uniflash.inc.mk
+PROGRAMMER ?= uniflash
+
+ifeq ($(PROGRAMMER),uniflash)
+  include $(RIOTMAKE)/tools/uniflash.inc.mk
+else ifeq ($(PROGRAMMER),openocd)
+  OPENOCD_CONFIG ?= $(RIOTBOARD)/common/cc26x2_cc13x2/dist/openocd.cfg
+  include $(RIOTMAKE)/tools/openocd.inc.mk
+endif
```

Comment:
You can also move this to `boards/common/cc26x2_cc13x2/Makefile.include` and then just `include $(RIOTBOARD)/common/cc26x2_cc13x2/Makefile.include`